src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/config/TokenConfig.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.sbl.sosialhjelp_mock_alt.config
+
+import no.nav.security.token.support.spring.test.MockOAuth2ServerAutoConfiguration
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+
+@Configuration
+@Import(MockOAuth2ServerAutoConfiguration::class)
+class TokenConfig {
+
+    // Mulig starte mockOAuth2Server manuelt fremfor med AutoConfiguration.
+    //
+    // LoginCookieAndTokenController
+    // - wellknown for issuer -> /login/metadata/{issuer}  -> gå til discoveryurl for issuer i mockOauth2Server
+    // - jwks for issuer      -> /login/jwks/{issuer}      -> gå til jwks url for issuer i mockOauth2Server
+    // - addCookie            -> /login/cookie/{issuer}    -> kopiert og tilpasset fra MockLoginController
```

Comment:
Starter dette automatisk når jeg kjører lokalt, eller må jeg gjøre noe?

Reply:
Stemmer, MockOAuth2ServerAutoConfiguration starter opp en MockOAuth2Server også når man kjører lokalt. Skal ikke være nødvendig å gjøre noe annerledes for lokal kjøring